Wrappers/Python/ccpi/optimisation/functions/MixedL21Norm.py

```diff
@@ -103,18 +73,7 @@ def convex_conjugate(self,x):
         """
         if not isinstance(x, BlockDataContainer):
             raise ValueError('__call__ expected BlockDataContainer, got {}'.format(type(x))) 
-                
-#        tmp1 = x.get_item(0) * 0.
-#        for el in x.containers:
-#            tmp1 += el.power(2.)
-#        tmp1.add(self.epsilon**2, out = tmp1)
-#        tmp = tmp1.sqrt().as_array().max() - 1
-#                    
-#        if tmp<=1e-6:
-#            return 0
-#        else:
-#            return np.inf            
-                
+                                        
         tmp = (x.pnorm(2).as_array().max() - 1)
```

Comment:
remove `as_array()` as `max` is a method of `DataContainer`

Reply:
done